packages/@lwc/engine-core/src/framework/base-lightning-element.ts

```diff
@@ -480,84 +534,26 @@ LightningElement.prototype = {
 
 const queryAndChildGetterDescriptors: PropertyDescriptorMap = create(null);
 
-// The reason we don't just call `import * as renderer from '../renderer'` here is that the bundle size
-// is smaller if we reference each function individually. Otherwise Rollup will create one big frozen
-// object representing the renderer, with a lot of methods we don't actually need.
-const childGetters = [
-    'children',
-    'childNodes',
-    'firstChild',
-    'firstElementChild',
-    'lastChild',
-    'lastElementChild',
-] as const;
-
-function getChildGetter(methodName: typeof childGetters[number]) {
```

Comment:
Why not keep these similar to `queryMethods` below for consistency. Avoids this repetitive block
```js
        const vm = getAssociatedVM(this);
        const renderer = getRendererFromVM(vm);
        if (process.env.NODE_ENV !== 'production') {
            warnIfInvokedDuringConstruction(vm, 'children');
        }
```

Reply:
the problem is that they are renaming renderer methods... so it is not that trivial anymore. I made the decision to roll them back to the original implementation which was part of the prototype/